### ISSUE TITLE
Dark Mode setting does not actually put app in dark mode

### ISSUE THREAD

**Issue by 1bcb** [paolostivanin/OTPClient#244]
OTPClient version: 2.5.0 from Flathub
Flatpak version: 1.12.2
Distro: Linux Mint Cinnamon 20.3.
Steps to reproduce: 
1)Launch OTP Client
2)Open Settings
3)Toggle "Dark Theme Enabled"

Expected results: The app interface changes to have a dark background a light text
Actual results: No change to the interface color

Log file from launching OTPClient from the terminal:
`flatpak run com.github.paolostivanin.OTPClient 
F: Ignoring D-Conf migrate-path setting /com/github/paolostivanin/OTPClient
Gtk-Message: 18:59:29.295: Failed to load module "xapp-gtk3-module"`
No additional text appears on the terminal when I attempt to change the app theme.

**Comment by paolostivanin** [comment]
Hello, thanks for reporting this.
The feature works fine on "standard" installation, so I guess it must be something related to how flatpak works. I'll look into it.

**Comment by paolostivanin** [comment]
Can you please check whether something like this helps?

https://ubuntuhandbook.org/index.php/2021/10/enable-dark-flatpak-apps-ubuntu-linux-mint/

**Comment by 1bcb** [comment]
That worked.  I installed `org.gtk.Gtk3theme.Mint-Y-Dark` (the same theme my host system is using), and it automatically switched OTPClient to dark mode.  

The "enable dark mode" setting in OTPClient still does nothing: even though I have both a light and dark theme installed as flatpaks, turning off the "enable dark mode" switch doesn't change OTPClient back to light mode.  That isn't a problem for me because I always want dark mode, and I can change it in Cinnamon's settings if I needed to for some reason.
